test/rules-google-event-measurement-fullstory.spec.ts

```diff
@@ -61,6 +61,17 @@ describe('Ruleset: Google Analytics Event Measurement to FullStory', () => {
         // now check that no calls get queued
         expect(await testHarness.popEvent(500)).to.be.undefined;
       });
+
+      it('gracefully handles a "clear" event', async () => {
+        await testHarness.execute(() => {
+          globalThis.dataLayer.push(
+            null,
+          );
+        });
+
+        const event = await testHarness.popEvent();
```

Comment:
Nit: Similar suggestion of potentially constraining to a 500ms timeout via `testHarness.popEvent(500);`. This keeps our test feedback loop a little shorter.

As I type this, I'm wondering about reducing our default timeout to 500ms (currently 1000ms). I'm happy to take that on as a TODO in a separate scope of work.

Reply:
Sure, or something you might use is `1.5 * DataHandler.DefaultDebounceTime` (`1.5` is illustrative, use `2` for comfort).  Not sure if we can access the class member `debounce` of `DataHandler` from tests, but that would be the more accurate debounce timeout.